package/MDAnalysis/coordinates/base.py

```diff
@@ -1589,6 +1608,24 @@ class Writer(IObase):
     See Trajectory API definition in :mod:`MDAnalysis.coordinates.__init__` for
     the required attributes and methods.
     """
+    class __metaclass__(type):
+        # Auto register upon class creation
+        def __init__(cls, name, bases, classdict):
+            type.__init__(type, name, bases, classdict)
```

Comment:
Is this the right form to use for calling the base metaclass's (in this case `type`) init? Does `super` work here, and should it be used?